app/src/main/res/values/themes.xml

```diff
@@ -36,15 +37,22 @@
     <item name="buttonStrokeColor">#5E072C</item>
   </style>
 
+  <style name="LondonThemeChip">
+    <item name="chipSelectedBackgroundColor">?colorPrimary</item>
```

Comment:
we can call it only `chipColor` as there are no other attributes

Reply:
We follow this pattern everywhere else so it's better we keep it consistent, see: https://github.com/Skyscanner/backpack/blob/master/decisions/theme-props-naming.md